pages/data.md

```diff
@@ -1,35 +1,60 @@
+# Beetles as Sentinel Taxa: Data
 
-This challenge uses data compiled from various [Zenodo records of the Butterfly Genetics Group at University of Cambridge](https://zenodo.org/communities/butterfly/records?q=&f=subject%3ACambridge&l=list&p=1&s=10&sort=newest). All data is licensed under [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/); see [butterfly_anomaly.bib](https://github.com/Imageomics/HDR-anomaly-challenge/blob/main/butterfly_anomaly.bib) for full citations.
+This challenge uses images and metadata associated with carabid beetle specimens collected by the [National Ecological Observatory Network (NEON)](https://www.neonscience.org/) for the "Ground beetles sampled from pitfall traps" data product (DP1.10022.001, [RELEASE-2025](https://doi.org/10.48443/cd21-q875)) to predict drought conditions at the location and time of sampling. 
+
+For this challenge, drought was characterized for each beetle sampling event using values of the Standardized Precipitation Evapotranspiration Index (SPEI) retrieved from the [GRIDMET Drought image collection](https://developers.google.com/earth-engine/datasets/catalog/GRIDMET_DROUGHT#description) (see [Abatzoglou (2011)](https://doi.org/10.1002/joc.3413).
+
+All data are licensed under <<get NEON preferred license infromation from Christine>>.
+
+## General Overview of Inputs and Expected Outputs
+
+**Model inputs** will be a flat metadata table (csv format) where each row is a metadata record for a specimen image. Multiple images will be associated with a sampling `event_id`, where a "sampleing event" corresponds with a location (NEON Site ID) and the date on which the beetle specimens were collected. Thus, multiple input records (multiple images of beetle specimens) will be used to predict a single SPEI value at a given site on a given date. Note that `event_id`s will be anonymized in the test dataset, but the metadata will include `site_id` and `collect_date` in the training data. 
+
+**What to predict:** The SPEI metrics represent cumulative drought conditions over a time window. For this challenge, participants will submit predictions for SPEI metrics representing three different time scales: 
+- `spei_30d` is drought conditions calcualted from data for the 30 day window preceding sample collection
+- `spei_1y` is the drought condition calculated from the year preceding sample collection
+- `spei_2y` is the drought condition calculated from the two years preceding sample collection
+
+**The submission file** will be a long-format flat table (csv file format) with a prediction for each `spei_30d`, `spei_1y`, and `spei_2y` for each `event_id`. Predictions will include a measure of uncertainty, so each prediction will include `mu` (mean) and `sigma` (standard devation). See the example submissoin below for more details. 
 
 ## Instructions to Download Training Data
 
+<<needs updating for beetle ml process, once we have it developed>>
```

Comment:
Presumably this also needs to be updated before merging, @sokole? Otherwise, we should choose a more appropriate (for public consumption) placeholder.